en/docs/get-started/tutorials/create-your-first-event-triggered-integration.md

```diff
@@ -0,0 +1,122 @@
+# Create Your First Event-Triggered Integration with Integration Studio
+
+Consider a scenario where a developer has created an integration using [WSO2 Integration Studio](https://wso2.com/integration/integration-studio/) and wants to use it as an event-triggered integration by deploying it in Choreo. In this tutorial, you will learn how to do the following actions to address this requirement:
+
+- Create an event-triggered integration in Choreo by connecting the GitHub repository which has an integration developed in [WSO2 Integration Studio](https://wso2.com/integration/integration-studio/).
+- Deploy the integration component to the development environment in Choreo.
+- Test the integration component.
+
+For this tutorial, let's use a basic sample application which listens to a [RabbitMQ](https://www.rabbitmq.com/) topic and logs incoming messages.
+
+!!! tip "Before you begin!"
+    To follow this tutorial, you can utilize a sample integration created with [WSO2 Integration Studio](https://wso2.com/integration/integration-studio/). Simply fork the [choreo-examples](https://github.com/wso2/choreo-examples) repository on GitHub.
+
+In this tutorial, we will be using the [simple-rabbitmq-listener](https://github.com/wso2/choreo-examples/tree/main/ipaas/wso2-synapse/simple-rabbitmq-listener) project in the **choreo-sample** repo. 
+
+## Step 0: Get Ready for the Tutorial
+
+To complete this tutorial, you need to start a RabbitMQ server either on a server or locally. Once it's started, obtain the `username`, `hostname`, `password` and `vhost` from the RabbitMQ instance, which you will later assign as environment variables. 
```

Comment:
Do we need to specify a specific version of RabbitMQ or perhaps specify to use the latest?

Reply:
We decided not to specify the version. we can suggest to use the latest version.